pkg/apis/monitoring/v1alpha1/alertmanager_config_types.go

```diff
@@ -281,24 +281,33 @@ type DiscordConfig struct {
 	// Whether or not to notify about resolved alerts.
 	// +optional
 	SendResolved *bool `json:"sendResolved,omitempty"`
-
 	// The secret's key that contains the Discord webhook URL.
 	// The secret needs to be in the same namespace as the AlertmanagerConfig
 	// object and accessible by the Prometheus Operator.
 	// +required
 	APIURL v1.SecretKeySelector `json:"apiURL"`
-
 	// The template of the message's title.
 	// +optional
+	// +kubebuilder:validation:MinLength=1
 	Title *string `json:"title,omitempty"`
-
 	// The template of the message's body.
 	// +optional
+	// +kubebuilder:validation:MinLength=1
 	Message *string `json:"message,omitempty"`
-
 	// HTTP client configuration.
 	// +optional
 	HTTPConfig *HTTPConfig `json:"httpConfig,omitempty"`
+	// The template of the content's body.
+	// +optional
+	// +kubebuilder:validation:MinLength=1
+	Content *string `json:"content,omitempty"`
+	// +optional
+	// +kubebuilder:validation:MinLength=1
+	Username *string `json:"username,omitempty"`
+	// +optional
+	// +kubebuilder:validation:MinLength=1
+	// +kubebuilder:validation:Pattern:="^http(s)?://.+$"
+	AvatarURL *string `json:"avatarURL,omitempty"`
```

Comment:
we can reuse the URL type in fact
```suggestion
	AvatarURL *URL `json:"avatarURL,omitempty"`
```

Reply:
thanks, updated